Assets/Scripts/CPURodInputHandler.cs

```diff
@@ -0,0 +1,96 @@
+using UnityEngine;
+
+public class CPURodInputHandler : IRodInputHandler
+{
+    private Transform _ballTransform;
+    private Transform _rodTransform;
+    private Doll[] _dolls;

```

Comment:
この2つは変更しないので読み取り専用にしよう。
```suggestion
    private readonly RodController _rodController;
    private readonly Doll[] _dolls;
```